website/src/app/kb/deploy/clients/readme.mdx

```diff
@@ -28,13 +31,157 @@ below. If you find an exception, please
+  you **must unset** the key from the managed configuration.
+</Alert>
+
+| Key                       | Type      | Default Value                      | Description                                                                                                                                                                                                                                                                                        | Available on        | Available since |
+| ------------------------- | --------- | ---------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------- | --------------- |
+| `authURL`                 | `String`  | `https://app.firezone.dev`         | The base auth URL of the Firezone application to sign in to. The `accountSlug` will be appended to this to form the complete sign-in URL.                                                                                                                                                          | macOS, iOS, Windows | 1.5.0           |
+| `apiURL`                  | `String`  | `wss://api.firezone.dev`           | The WebSocket URL of the Firezone control plane.                                                                                                                                                                                                                                                   | macOS, iOS, Windows | 1.5.0           |
+| `logFilter`               | `String`  | `info`                             | The [`RUST_LOG`-formatted](https://rust-lang-nursery.github.io/rust-cookbook/development_tools/debugging/config_log.html) log filter to apply to the connectivity library logger. Increasing the log level here can help troubleshoot connectivity issues at the cost of increased log file sizes. | macOS, iOS, Windows | 1.5.0           |
+| `accountSlug`             | `String`  | `<empty>`                          | The account slug or ID of your Firezone account.                                                                                                                                                                                                                                                   | macOS, iOS, Windows | 1.5.0           |
+| `startOnLogin`            | `Boolean` | `false`                            | Whether the client should start automatically on login.                                                                                                                                                                                                                                            | macOS, iOS, Windows | 1.5.0           |
```

Comment:
Not available on Windows.

Reply:
Just double-checking, this is still the case right?

Reply:
Not as a managed config, no. To start a program on login, you configure dedicated autostart entries Intune, separate from the app config.